vignettes/art-contrasts.Rmd

```diff
@@ -196,26 +203,22 @@ subtracted out the effect of `C - D`. We can see this if we take the ART estimat
 ART estimate for `A,C - A,D` (`125.12`) here, we can get approximate an estimate of the difference (`-123.13 + 125.12 == 1.99`)
 that is consistent with the expected 0 (given the SE here).
 
-If we first combine the factors before aligning-and-ranking, we can get an estimate of these effects:
+The ART-C procedure however, was developed to align and rank data specifically for contrasts involving levels from any number of factors.
 
 ```{r}
-#DO NOT DO THIS WITHOUT READING THE NOTE BELOW
-df$X = with(df, X1:X2)
-m.art.12 = art(Y ~ X, data=df)
-contrast(emmeans(artlm(m.art.12, "X"), ~ X), method="pairwise")
+art.con(m.art, "X1:X2")
 ```
 
-Note that the above table is not in the same order as the previous ones, but does have similar t values to the corresponding effects
-from the linear model. __However, it is not clear this will generalize.__ In this case there aren't any other factors except these two, 
-so this is equivalent to just ranking the response before fitting the model, and in the general case it can incorrectly estimate 
-interactions (see issues with using the plain-old rank transform in the ART literature, specifically Higgins' papers). Thus, I 
-can't recommended this approach in the general case (or at least, not without more investigation).
+Like the linear model, art.con  correctly estimates the difference between `A,C - A,D` as `~0`. In fact, its results agree with the linear model for all contrasts conducted. (Note that the art.con and linear model results appear in a different order; specifically A C - A D appears second for the linear model and first for art.con)
+
+The syntax used above is consistent with *term* syntax used by artlm. art.con also accepts the *formula* syntax accepted by emmeans. We can conduct the same contrasts as above using the following syntax:
```

Comment:
Add formatting for functions (`artlm()`, etc). Also can be helpful to readers to visually indicate when talking about functions from other packages by using `::`, like: `emmeans::emmeans()`